core/iwasm/libraries/libc-wasi/libc_wasi_wrapper.c

```diff
@@ -969,9 +1007,48 @@ wasi_poll_oneoff(wasm_exec_env_t exec_env, const wasi_subscription_t *in,
         || !validate_native_addr(nevents_app, sizeof(uint32)))
         return (wasi_errno_t)-1;
 
-    err = wasmtime_ssp_poll_oneoff(curfds, in, out, nsubscriptions, &nevents);
-    if (err)
-        return err;
```

Comment:
The changes of this PR are only for multi-threading mode, can we keep the original implementation for non multi-threading mode? Or, use WASM_ENABLE_THREAD_MGR to control the code:
```C
#if WASM_ENABLE_THREAD_MGR == 0
    err = wasmtime_ssp_poll_oneoff(curfds, in, out, nsubscriptions, &nevents);
    if (err)
        return err;
#else
...
#endif
```

Reply:
done